### DISCUSSION TITLE
Mobile screen responsiveness

### DISCUSSION THREAD

**Discussion by dominicsavio3** [GrapesJS/grapesjs#4348] [Q&A]
Hi @artf 
    
   I am using grapesjs-preset-webpage plugin in my GrapesJS application. The default width of the mobile set to 320px. I just trying to change to 420px but it seems the default is always set to 320px is there any way to customize the mobile screen.
![Screenshot from 2022-05-26 19-16-40](https://user-images.githubusercontent.com/97028786/170500466-e9a7c7fc-6765-49a3-9bc0-b1a94a78a7c5.png)

**Comment by artf** [comment]
Try to check the [Devices API](https://grapesjs.com/docs/api/device_manager.html)
